.github/workflows/ci.yml

```diff
@@ -134,6 +134,10 @@ jobs:
         uses: dtolnay/rust-toolchain@1.72.1
         with:
           components: clippy
+      - name: Install scarb 
+        uses: software-mansion/setup-scarb@v1
+        with:
+          scarb-version: "2.4.0"
```

Comment:
I'd suggest we just use the `make` target. We'll eventually want to do proper scrutiny of external actions so being conservative about new ones sounds like a good idea to me.
@klaus993 @pefontana what do you think?

Reply:
The `deps` target installs scarb using the installation script, but I couldn't get it to work for the macos CI, thats why I had to add the actions. We can create an issue to remove the action once the installation is fixed